setup.py

```diff
@@ -70,6 +70,24 @@
     author_email="reingart@gmail.com",
     url="https://github.com/reingart/pyafipws",
     license="LGPL-3.0-or-later",
+    install_requires=[
+        "httplib2==0.9.2;python_version <= '2.7'",
+        "httplib2>=0.20.4;python_version > '3'",
+        "pysimplesoap==1.08.14;python_version <= '2.7'",
+        "pysimplesoap @ git+https://github.com/pysimplesoap/pysimplesoap.git@py311#pysimplesoap;",
```

Comment:
Here it is missing the python 3 conditional, right?
```suggestion
        "pysimplesoap @ git+https://github.com/pysimplesoap/pysimplesoap.git@py311#pysimplesoap; python_version > '3'",
```

Reply:
done